Jenkinsfile

```diff
@@ -81,10 +84,23 @@ for (int i = 0; i < splits.size(); i++) {
                     set-java.sh $javaVersion
                     eval \$(vnc.sh)
                     java -version
-                    run.sh firefox ${jenkinsUnderTest} -Dmaven.repo.local=${WORKSPACE_TMP}/m2repo -Dmaven.test.failure.ignore=true -DforkCount=1 -B
+                    run.sh ${browser} ${jenkinsUnderTest} -Dmaven.repo.local=${WORKSPACE_TMP}/m2repo -Dmaven.test.failure.ignore=true -DforkCount=1 -B
+                    cp --verbose target/surefire-reports/TEST-*.xml /reports
                 """
               }
             }
+            launchable.install()
+            withCredentials([string(credentialsId: 'launchable-jenkins-acceptance-test-harness', variable: 'LAUNCHABLE_TOKEN')]) {
+              launchable('verify')
+              /*
+               * TODO Create a Launchable build and session earlier, and replace "--no-build" with
+               * "--session" to associate these test results with a particular build. The commits
+               * associated with the Launchable build should be the commits of the transitive closure of
+               * the Jenkins WAR under test in this build as well as the commits of the transitive closure
+               * of the ATH JAR.
+               */
```

Comment:
A bit of foreshadowing as to what is involved in the next step, commit collection. At present, this repository only pins the Jenkins core version, not plugin versions, so the relevant commits are the transitive closure of the core under test and the ATH under test. Collecting this with mathematical precision will require creating a new tool, but for the short term I plan on at least feeding Launchable the commits from Jenkins core itself (omitting its transitive dependencies like Stapler) and ATH itself (omitting its transitive dependencies like Remoting) since that is easy to implement (see my other PRs to `pom` for example). (Anything less than that, and the model would be so deprived of useful data that it would be pointless.) The idea is that this prototype will take shape incrementally, the model becoming more accurate as more and more of the design document is implemented. Obviously a fully transitive dependency analysis is needed to give the model the highest degree of confidence, but we don't have to wait for that to begin training the model with a fair degree of confidence.